scripts/zones/Newton_Movalpolos/mobs/Goblin_Collector.lua

```diff
@@ -0,0 +1,27 @@
+------------------------------
+-- Area: Newton Movalpolos
+--   NM: Goblin Collector
+------------------------------
+require("scripts/globals/status")
+------------------------------
+
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMod(xi.mod.MATT, 150)
+    mob:setMod(xi.mod.BINDRES, 40)
+    mob:setMod(xi.mod.GRAVITYRES, 40)
+end
+
+entity.onMobFight = function(mob, target)
+    -- Resets threat on every auto attack
+    mob:addListener("ATTACK","COLLECTOR_ATTACK", function(mob)
+        mob:resetEnmity(target)
+    end)
```

Comment:
From CI, you are " shadowing upvalue argument mob on line 16"
```
    mob:addListener("ATTACK","COLLECTOR_ATTACK", function(goblin)
        goblin:resetEnmity(target)
    end)
```
Use this instead, or change "goblin" to whatever else you see fit